YouTubeDeepDarkMaterial.user.css

```diff
@@ -112,12 +113,12 @@
 			--shadow: 0 1px 0.5px rgba(0, 0, 0, .13);
 		}
 		else if stylus-deepdark-style == "Firefox-Dark" {
-			--main-color: #5675b9;
-			--main-background: #272b35;
-			--second-background: #181d20;
-			--hover-background: #353a44;
-			--main-text: #e3eef9;
-			--dimmer-text: #bec0cc;
+			--main-color: #00ddff;
+			--main-background: #1c1b22;
+			--second-background: #0e0d11;
```

Comment:
Swap colors around, so that highlights can have lighter colors than the background.